Dockerfile

```diff
@@ -9,10 +9,8 @@
 # either express or implied. See the License for the specific language governing permissions
 # and limitations under the License.
 # ----------------------------------------------------------------------------------------
-FROM azul/zulu-openjdk-alpine:11 as builder
-MAINTAINER dev@sling.apache.org
```

Comment:
Did you intentionally drop the maintainer line?

Reply:
Ah, yes I did, but I meant to replace it with the appropriate OpenContainer labels. Fixed in: https://github.com/apache/sling-org-apache-sling-committer-cli/pull/12/commits/12952a66be3d968befa2bbef53f47b0c57b952ed